src/WarcraftLegacies.Source/Quests/Gilneas/QuestCrowley.cs

```diff
@@ -0,0 +1,33 @@
+﻿using MacroTools.ObjectiveSystem.Objectives.TimeBased;
+using MacroTools.QuestSystem;
+using static War3Api.Common;
+
+namespace WarcraftLegacies.Source.Quests.Gilneas
+{
+  /// <summary>
+  /// Darius gets released from Prison " 
+  /// </summary>
+  public sealed class QuestCrowley : QuestData
+  {
+    private const int AltarId = Constants.UNIT_H02X_ALTAR_OF_KINGS_GILNEAS_ALTAR;
+
+    /// <summary>
+    /// IniInitializes a new instance of the <see cref="QuestCrowley"/> class.
+    /// </summary>
+    public QuestCrowley() : base("The Rebel",
+      "Darius Crowley has been imprisoned since the Northgate rebellion, if Gilneas were to be in grave peril an early release might be the solution.",
```

Comment:
```suggestion
      "Darius Crowley has been imprisoned since the Northgate rebellion. If Gilneas were to fall into grave peril, an early release might be necessary.",
```